deploy/contracts.js

```diff
@@ -30,6 +20,7 @@ module.exports = async () => {
         "0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a",
         "0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6",
     ];
+    const operatorPK = "0xf214f2b2cd398c806f84e317254e0f0b801d0643303237d97a22a48e01628897";
```

Comment:
?

Reply:
That's a default operator private key take from the default hardhat signers[10], that might be required at some point